server/services/Jimp.js

```diff
@@ -0,0 +1,10 @@
+const Jimp = require('jimp');
+
+ //crop img
+ module.exports = async (file,width,height) => {
+    const crop = await Jimp.read(file);
+    crop
+    .resize(width,height) //size
+    .quality(80)          //quality
+    .write(`static/uploads/avatars/small${+ new Date()}.${crop.getExtension()}`); //save
```

Comment:
As Sasha told us, better save it to .png format, it is best for web. Propose to save it to .png Seems Jimp supports it - @jimp/png

Reply:
These are the old changes
I deleted this library because you cropped the image on the client side